src/Drivers/Decorator.php

```diff
@@ -488,6 +499,27 @@ public function name($feature)
         return $this->resolveFeature($feature);
     }
 
+    /**
+     * Retrieve the feature's class.
+     *
+     * @param  string  $feature
+     * @return mixed
+     */
+    public function instance($name)
+    {
+        $feature = $this->nameMap[$name] ?? $name;
+
+        if (is_string($feature) && class_exists($feature)) {
```

Comment:
It would be amazing if we could add an enum check here as well 😄

Reply:
What would that do, @cosmastech?

Reply:
If I misunderstood this, disregard:

Would love to have the ability to call `Feature::define(FeatureFlagEnum::NewFeature, fn() => false)`, but maybe that's not really what this PR is aiming for.